crest/Assets/Crest/Crest/Scripts/OceanBuilder.cs

```diff
@@ -211,24 +211,22 @@ public static void GenerateMesh(OceanRenderer ocean, int lodDataResolution, int
             for (int i = 0; i < ocean.transform.childCount; i++)
             {
                 var child = ocean.transform.GetChild(i);
-                if (child.name.StartsWith("LOD"))
+                if (child.name.StartsWith("Tile_L"))
                 {
                     child.parent = null;
                     Object.Destroy(child.gameObject);
                     i--;
                 }
             }
 
-            int startLevel = 0;
-            for( int i = 0; i < lodCount; i++ )
+            for (int i = 0; i < lodCount; i++)
             {
                 bool biggestLOD = i == lodCount - 1;
-                GameObject nextLod = CreateLOD(ocean, i, lodCount, biggestLOD, meshInsts, lodDataResolution, geoDownSampleFactor, oceanLayer);
-                nextLod.transform.parent = ocean.transform;
+                //nextLod.transform.parent = ocean.transform;
 
                 // scale only horizontally, otherwise culling bounding box will be scaled up in y
-                float horizScale = Mathf.Pow( 2f, (float)(i + startLevel) );
-                nextLod.transform.localScale = new Vector3( horizScale, 1f, horizScale );
+                //nextLod.transform.localScale = new Vector3(horizScale, 1f, horizScale);
```

Comment:
Is there any particular reason why this is being commented-out instead of being deleted?

Reply:
There's another similar line above this section as well which is commmented out.